google-cloud-api_gateway-v1/synth.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import synthtool.languages.ruby as ruby
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "apigateway", "v1",
+    extra_proto_files=[
+        "google/cloud/common_resources.proto",
+    ],
+    generator_args={
+        "ruby-cloud-gem-name": "google-cloud-api_gateway-v1",
+        "ruby-cloud-title": "API Gateway V1",
+        "ruby-cloud-description": "API Gateway enables you to provide secure access to your backend services through a well-defined REST API that is consistent across all of your services, regardless of the service implementation. Clients consume your REST APIS to implement standalone apps for a mobile device or tablet, through apps running in a browser, or through any other type of app that can make a request to an HTTP endpoint.",
+        "ruby-cloud-env-prefix": "API_GATEWAY",
+        "ruby-cloud-grpc-service-config": "google/cloud/apigateway/v1/apigateway_grpc_service_config.json",
+        "ruby-cloud-namespace-override": "ApiGateway=APIGateway",
```

Comment:
Are you sure you want `API` in full caps in the module name `APIGateway` but not in the module name `ApiGatewayService`?

Reply:
We're re-evaluating this...

Reply:
Changed it back to `ApiGateway`.